sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelPool.java

```diff
@@ -877,6 +900,7 @@ private double computeLoadFactor() {
     private void doAcquireChannel(final ChannelPromiseWithExpiryTime promise, final Channel candidate) {
         this.ensureInEventLoop();
         acquiredChannels.put(candidate, candidate);
+        totalAcquiredChannels.incrementAndGet();
```

Comment:
for the totalAcquireChannels -> are we trying to track how many times we opened a connection? then the right place to track should be in `notifyChannelConnect`.   `doAcquireChannel ` will be called for every request regardless whether this is a new established channel or reuse an existing one

Reply:
Fixed